### PR TITLE
Get rid of the Newsletter page Link In contributing/community.md

### LINKED ISSUE #21809: chore(www): Get rid of the Newsletter page
## Summary

Get rid of https://gatsbyjs.org/newsletter, and replace the only link to it (in contributing/community.md) with the email capture form component.

### Motivation

It's one less page that needs to be translated, it's only linked to in one place and can be replaced with a more direct action.

### PR DESCRIPTION
**Pull request by saadpasta**
## Description

Remove  https://gatsbyjs.org/newsletter link in (contributing/community.md) with Email Form Component 

## Related Issues
Fixes #21809

### CODE REVIEW

File: docs/contributing/community.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The Gatsby community welcomes contributions. Please refer to the guides below on
 
 Sign up for the Gatsby newsletter to keep up with the latest from the Gatsby community! Hear about new features, tips & tricks, and what people are building.
 
-[Sign up here](/newsletter/)
+<EmailCaptureForm signupMessage="Want to keep up with the latest tips &amp; tricks? Subscribe to our newsletter!" />
```
Comment (tesseralis) [inline docs/contributing/community.md]: We'd also need to get rid of `www/src/pages/newsletter.js`.

**Comment by AishaBlake** [comment]
This would also need a [redirect](https://www.gatsbyjs.org/contributing/docs-contributions/#docs-renaming-instructions) to the Community page. I've checked in with the Marketing to make sure this won't adversely affect anything on their end. Will post again when I have a response!

File: docs/contributing/community.md (hunk 1 above)
Comment (saadpasta) [reply docs/contributing/community.md]: should I remove that file?

Comment (tesseralis) [reply docs/contributing/community.md]: yup!

Comment (saadpasta) [reply docs/contributing/community.md]: done

**Comment by AishaBlake** [comment]
> This would also need a [redirect](https://www.gatsbyjs.org/contributing/docs-contributions/#docs-renaming-instructions) to the Community page. I've checked in with the Marketing team to make sure this won't adversely affect anything on their end. Will post again when I have a response!

@saadpasta, could you add the redirect as well? I got the feedback from Marketing and this should be good to go after that's added.

**Comment by saadpasta** [comment]
@AishaBlake Can you have a look if the redirect URL is correct

File: www/redirects.yaml [hunk 2]
```diff
@@ -126,6 +126,8 @@
   toPath: /docs/data-fetching/
 - fromPath: /docs/centralizing-your-sites-navigation/
   toPath: /docs/creating-dynamic-navigation/
+- fromPath: /newsletter/
+  toPath: /contributing/community/#newsletter
```
Comment (tesseralis) [inline www/redirects.yaml]: @AishaBlake I'm not sure what the right page for this is, but I'm pretty sure hashes don't work in redirects?

Comment (saadpasta) [reply www/redirects.yaml]: @AishaBlake can you please review so I can change the redirect url.

**Comment by saadpasta** [comment]
Thanks @AishaBlake.

**Pull request merged**